From 4df8f031643d949d72ef091396728ea5eab5cf07 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Wed, 1 Jun 2016 12:22:13 +0200
Subject: [PATCH] ARM: bcm2835: Give HDMI a HPD GPIO on the pi3
Git-commit: 4df8f031643d949d72ef091396728ea5eab5cf07
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: git://git.kraxel.org/linux
References: FATE#319481

This is the wrong GPIO, but it means vc4 probes and turns on hdmi.

[ kraxel: flip from active low to active high ]

Acked-by: Michal Marek <mmarek@suse.cz>
---
 arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts b/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
index 6f47dd2bb1db..e2c3801dd6ae 100644
--- a/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
@@ -28,3 +28,7 @@
 &uart1 {
 	status = "okay";
 };
+
+&hdmi {
+	hpd-gpios = <&gpio 46 GPIO_ACTIVE_HIGH>;
+};
-- 
1.8.5.6

